jenkins-integration/scripts/foss_install.sh

```diff
@@ -1,6 +1,22 @@
 #!/bin/bash
 set -x
 
+## Requires:
+## 1. Version of puppet-agent package to install
+##    Can be specified with either:
+##    - ENV['PUPPET_BUILD_VERSION']=1.3.0
+##    - hosts.yaml#CONFIG#puppet_version: 1.3.0
+##
+## 2. Version of puppet-server package to install
+##    Can be specified with either:
+##    - ENV['PUPPETSERVER_BUILD_VERSION']=2.2.1
+##    - hosts.yaml#CONFIG#puppetserver_version=1.3.0
+
+if [ -z $BEAKER_CONFIG ]; then
+    echo "BEAKER_CONFIG hosts configuration required"
+    exit 1
+fi
```

Comment:
It's not, but during development I run this file directly, and having this fail-fast is useful.